src/main.py

```diff
@@ -8,24 +8,63 @@
 COL_SIZE = 64
 
 config = hub75.Hub75SpiConfiguration()
+##-----------------------------------------------------------------------------
+## row select pins
+config.line_select_a_pin_number = 15
```

Comment:
I don't see a way to merge a pull request without merging all of the changes. Could you revert this to the version of main.py on the main branch?

Reply:
Yes, of course, no problem! I'm an absolute beginner with Git(Hub), please excuse.

Reply:
Should be OK now, please check!